csm/plugins/cortx/rgw.py

```diff
@@ -115,22 +111,19 @@ def _supress_response_keys(self, operation, response) -> Any:
             operation (str): IAM operation.
             response (dict): Response from s3 server.
 
-        Raises:
-            CsmInternalError
-
         Returns:
             Modified response.
         """
         suppressed_response = response
         keys = self._api_suppress_payload_schema.get(operation)
-        Log.debug(f"Suppressing {keys} keys from raw response.")
         if keys:
+            Log.info(f"Suppressing {keys} keys from raw response.")
```

Comment:
No need to log the internals of the logic

Reply:
removed {keys}.